packages/cli/src/cli/provider/action.provider.tag.ts

```diff
@@ -48,10 +48,10 @@ export class ProviderUpdateTagAction extends CommandLineAction {
             }
         }
 
-        if (before !== null) {
+        if (before != null) {
```

Comment:
Isn't this should updated to `!==`

Reply:
`== null` allows us to match `before === null || before === undefined`  its just simpler for most of the time where you dont care if its a null or undefined value